addons/mail/static/src/models/thread.js

```diff
@@ -1059,6 +1041,13 @@ registerModel({
                 });
             return replace(allAttachments);
         },
+        /**
+         * @private
+         * @returns {FieldCommand}
+         */
+        _computeChannel() {
+            return this.model === 'mail.channel' ? insertAndReplace({ id: this.id }) : clear();
```

Comment:
Just a comment to point out the problematic of having a single and unique identification in models.
I would expect here just `insertAndReplace()`, without passing any data. But because of single and unique identification, we must keep a single one... and it makes things inappropriately technical and cumbersome, like here.

Reply:
We don't really have the choice right now since we need thread and channel model to live side by side for a while.

Reply:
Yes, I fully understand. As I said, just a comment to, in a way, point out the current design issue with unique identification here, and justify why it's relevant to support multiple identifications in the future.